.github/workflows/ci.yml

```diff
@@ -1,10 +1,7 @@
-name: Run tests
```

Comment:
How come we removed the name? Not needed?

Reply:
Yeah. We've broadly decided on no name in the conventions doc (see the link in the PR description).